catalog/admin.py

```diff
@@ -1,3 +1,32 @@
 from django.contrib import admin
+from .models import Author, Genre, Book, BookInstance
 
-# Register your models here.
+
+class AuthorAdmin(admin.ModelAdmin):
+    list_display = ("last_name", "first_name", "date_of_birth", "date_of_death")
```

Comment:
Hãy phân biệt list_display và fields?

Reply:
#### `list_display` :
- Dùng để kiểm soát trường nào được hiển thị trên trang danh sách của quản trị viên (admin page)
- Nếu không sử dụng list_display, trang admin page sẽ hiển thị một cột trong đó biểu diễn *__str __*  của từng đối tượng. *(__str__() representation)*
- Đặc biệt, với trường hợp quan hệ liên kết là *ManyToManyField (n to n)* sẽ không được hỗ trợ hiển thị vì điều đó sẽ đòi hỏi phải thực thi một câu lệnh SQL riêng cho mỗi hàng trong bảng, mà chi phí để query với trường hợp này là lớn thậm chí rất lớn *(Django prevents this because there would be a large database access "cost" in doing so)*
#### `fields`:
- Dùng để tùy chọn các trường để thực hiện các thay đổi bố cục đơn giản trong các hiển thị trên trang admin page, chẳng hạn như chỉ hiển thị một tập hợp con các trường có sẵn, sửa đổi thứ tự của chúng hoặc nhóm chúng thành các hàng. 
- Các trường được hiển thị mặc định theo chiều dọc, nhưng sẽ hiển thị theo chiều ngang nếu ta nhóm chúng vào 1 tuple.